docs/learning-powershell/using-vscode.md

```diff
@@ -73,56 +73,91 @@ you will need to add a new variable to your user settings file.
 
 Debugging with Visual Studio Code
 ----
+### No-workspace debugging
+As of Visual Studio Code version 1.9 you can debug PowerShell scripts without having to open the folder containing the PowerShell script.
+Simply open the PowerShell script file with **File->Open File...**, set a breakpoint on a line (press F9) and then press F5 to start debugging.
+You will see the Debug actions pane appear which allows you to break into the debugger, step, resume and stop debugging.
+
+### Workspace debugging
+Workspace debugging refers to debugging in the context of a folder that you have opened in Visual Studio Code using **Open Folder...** from the **File** menu.
+The folder you open is typically your PowerShell project folder and/or the root of your Git repository.
+
+Even in this mode, you can start debugging the currently selected PowerShell script by simply pressing F5.
+However, workspace debugging allows you to define multiple debug configurations other than just debugging the currently open file.
+For instance, you can add a configurations to:
+
+* Launch Pester tests in the debugger
+* Launch a specific file with arguments in the debugger
+* Launch an interactive session in the debugger
+* Attach the debugger to a PowerShell host process
+
+Follow these steps to create your debug configuration file:
+1. Open the **Debug** view by pressing **Ctrl+Shift+D** (**Cmd+Shift+D** on Mac).
+2. Press the **Configure** gear icon in the toolbar.
+3. Visual Studio Code will prompt you to **Select Environment**.
+Choose **PowerShell**.
 
--	Open a file folder (**File->Open Folder**) that contains the PowerShell modules or scripts you have written already and want to debug.
-In this example, we saved the helloworld.ps1 under a directory called "demo".
-Thus we select the "demo" folder and open it in Visual Studio Code.
-
--	Creating the Debug Configuration (launch.json)
-
-  Because some information regarding your scripts is needed for debugger to start executing your script, we need to set up the debug config first.
-  This is one-time process to debug PowerShell scripts under your current folder.
-  In our case, the "demo" folder.
+   When you do this, Visual Studio Code creates a directory and a file **.vscode\launch.json** in the root of your workspace folder.
```

Comment:
Genereally, it seems file paths are in quotes, so if you put this in quotes, I think the spellchecker will be fine with it